move newer image into centos/rhscl namespace

praiskup opened the pull request:
While we are on it, also unify the build scripts among images'
repositories:

- hack/ directory should be _just_ mirrored
- Makefile should be as declarative as possible
- build.sh should not rely on repository name

praiskup commented:
@hhorak fyi

hhorak commented:
[test]

mfojtik commented:
@praiskup can you address the comments @bparees made https://github.com/openshift/mysql/pull/98 here (and in other PR's?) Also we need to coordinate this as we have to update the Jenkins push images job to pickup new namespaces.

praiskup commented:
@mfojtik, I believe I already addressed those comments (because the pull request has
already been merged).  This pull-request should be in sync with the others.

bparees commented on Makefile:
```diff
@@ -1,26 +1,7 @@
+# Variables are documented in hack/build.sh.
+BASE_IMAGE_NAME = python
+VERSIONS = 2.7 3.3 3.4
+OPENSHIFT_NAMESPACES = 2.7 3.3
```
why isn't 2.7 going to centos/rhscl?  openshift doesn't own that image.

praiskup commented:
@bparees wrote:
> why isn't 2.7 going to centos/rhscl? openshift doesn't own that image.
I didn't know that.  PTAL.

hhorak commented:
[test]

bparees commented:
[test]
